tools/docker/client.ini

```diff
@@ -0,0 +1,40 @@
+[blockstack-client]
+protocol = http
+api_endpoint_host = 0.0.0.0
+poll_interval = 300
+api_endpoint_port = 6270
+api_password = dummywalletpassword
+metadata = metadata
+server = blockstack-core
+email =
+api_endpoint_bind = 0.0.0.0
+blockchain_writer = blockstack_utxo
+queue_path = /root/.blockstack/queues.db
+storage_drivers = disk,http,dht
+blockchain_reader = blockstack_utxo
+client_version = 0.18.0.0
+storage_drivers_required_write = disk,dropbox
+port = 6264
+anonymous_statistics = True
+
+[blockchain-reader]
+url = https://utxo.blockstack.org
+utxo_provider = blockstack_utxo
+
+[subdomain-resolution]
+subdomains_db = /root/.blockstack/subdomains.db
+resolving_subdomains = True
+
+[bitcoind]
+passwd = blockstacksystem
+regtest = False
+spv_path = /root/.virtualchain-spv-headers.dat
+server = 172.16.8.8
```

Comment:
Which server is this?  Does docker-compose instantiate a bitcoind node at this address? Should this be bitcoin.blockstack.com?